app/assets/javascripts/drafts.coffee

```diff
@@ -76,6 +76,11 @@ $(document).ready ->
     e.stopImmediatePropagation()
 
   incrementElementIndex = (newDiv, multipleIndex, simple, type) ->
```

Comment:
It looks like point 2 above is incrementing properly now, but I think point 1 ('instrument_n_characteristics') is still not incrementing correctly:
![screen shot 2017-04-12 at 4 56 18 pm](https://cloud.githubusercontent.com/assets/1174907/24979110/0c497874-1fa1-11e7-9634-0b17de33b947.png)
and
![screen shot 2017-04-12 at 4 59 43 pm](https://cloud.githubusercontent.com/assets/1174907/24979259/83492b18-1fa1-11e7-83f0-ca2ce45b03ff.png)